src/main/java/frc/robot/commands/VisionCommands.java

```diff
@@ -0,0 +1,87 @@
+public class VisionCommands {
+  /**
+   * Returns a command to use AllignToReef to allign to a reef position and score coral at Level 1.
+   *
+   * @param subsystems The Subsystems Container.
+   * @param reefPosition The Specified reef position.
+   * @return
+   */
+  public static Command AutoScoreToL1(Subsystems subsystems, ReefPosition reefPosition) {
+    return Commands.sequence(
+        Commands.parallel(
+            DriveCommands.alignToReefPosition(subsystems, reefPosition),
+            CoralAndElevatorCommands.raiseElevatorAndCoralArm(subsystems, ElevatorLevel.L1)),
+        CoralCommands.outtakeUntilCoralNotDetected(subsystems),
+        ElevatorCommands.stowElevatorAndArmForCoral(subsystems));
+  }
+
+  /**
+   * Returns a command to use AllignToReef to allign to a reef position and score coral at Level 2.
+   *
+   * @param subsystems Subsystems container.
+   * @param reefPosition Specified reef Position.
+   * @return
+   */
+  public static Command AutoScoreToL2(Subsystems subsystems, ReefPosition reefPosition) {
+    return Commands.sequence(
+        Commands.parallel(
+            DriveCommands.alignToReefPosition(subsystems, reefPosition),
+            CoralAndElevatorCommands.raiseElevatorAndCoralArm(subsystems, ElevatorLevel.L2)),
+        CoralCommands.outtakeUntilCoralNotDetected(subsystems),
+        ElevatorCommands.stowElevatorAndArmForCoral(subsystems));
+  }
+
+  /**
+   * Returns a command to use AllignToReef to allign to a reef position and score coral at Level 3.
+   *
+   * @param subsystems The Subsystems Container.
+   * @param reefPosition Specified Reef Position.
+   * @return
+   */
+  public static Command AutoScoreToL3(Subsystems subsystems, ReefPosition reefPosition) {
+    return Commands.sequence(
+        Commands.parallel(
+            DriveCommands.alignToReefPosition(subsystems, reefPosition),
+            CoralAndElevatorCommands.raiseElevatorAndCoralArm(subsystems, ElevatorLevel.L3)),
+        CoralCommands.outtakeUntilCoralNotDetected(subsystems),
+        ElevatorCommands.stowElevatorAndArmForCoral(subsystems));
+  }
+
+  /**
+   * Returns a command to use AllignToReef to allign to a reef position and score coral at Level 4.
+   *
+   * @param subsystems Subsystems Container.
+   * @param reefPosition Specified Reef Position.
+   * @return
+   */
+  public static Command AutoScoreToL4(Subsystems subsystems, ReefPosition reefPosition) {
+    return Commands.sequence(
+        Commands.parallel(
+            DriveCommands.alignToReefPosition(subsystems, reefPosition),
+            CoralAndElevatorCommands.raiseElevatorAndCoralArm(subsystems, ElevatorLevel.L4)),
+        CoralCommands.outtakeUntilCoralNotDetected(subsystems),
+        ElevatorCommands.stowElevatorAndArmForCoral(subsystems));
+  }
+
+  public static Command AutoIntakeToCoralStation(Subsystems subsystems) {
```

Comment:
Use lower camel-case for method names.
```suggestion
  public static Command autoIntakeFromCoralStation(Subsystems subsystems) {
```